### DISCUSSION TITLE
Background images

### DISCUSSION THREAD

**Discussion by tonirc** [qmd-lab/closeread#83] [Ideas]
First of all, closeread is fantastic! Thanks :) 
Can we have different background images as you scroll?

**Comment by andrewpbray** [comment]
Thanks @tonirc !

Hmm, yes, that should be possible. If you use one of the `overlay` layouts, the sticky takes up the full width of the screen. So you _should_ be able to just declare an image a sticky and it should work. Could you give it a try and report back with a link to your source and document?

It's possible you might bump into this issues: #81 ...
